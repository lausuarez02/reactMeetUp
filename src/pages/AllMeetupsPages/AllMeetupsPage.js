//components
import MeetupItem from "components/meetups/MeetupItem/MeetupItem";
//utils
import classes from "components/meetups/MeetupList.module.css";


export default function AllMeetupsPage() {
  return (
    <section>
      <h1>All Meetups</h1>
      <ul className={classes.list}>
        <MeetupItem />
      </ul>
    </section>
  );
}
