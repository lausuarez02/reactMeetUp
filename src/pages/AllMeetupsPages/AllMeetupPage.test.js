import { render } from "@testing-library/react";
import AllMeetupsPage from "./AllMeetupsPage";
//utils
import { Provider } from 'react-redux';

import {createStore} from 'redux';
import {BrowserRouter} from 'react-router-dom'


const mockedUsedNavigate = jest.fn();

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useNavigate: () => mockedUsedNavigate
}))

describe("AllMeetupsPage", () => {
      let store;
  beforeEach(() => {
      store = createStore((state = {}, action) => {
        return state
      });
  })
  test("renders correctly", () => {
    const { asFragment } = render(
        <Provider store={store}>
        <BrowserRouter>
          <AllMeetupsPage 
            
          />
        </BrowserRouter>
      </Provider>);
    expect(asFragment()).toMatchSnapshot();
  });
});
