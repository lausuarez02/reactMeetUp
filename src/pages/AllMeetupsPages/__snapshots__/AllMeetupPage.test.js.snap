// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllMeetupsPage renders correctly 1`] = `
<DocumentFragment>
  <section>
    <h1>
      All Meetups
    </h1>
    <ul
      class="list"
    >
      <p
        data-testid="component"
      >
        Loading...
      </p>
    </ul>
  </section>
</DocumentFragment>
`;
